resources/backup.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: stolon-backup
+spec:
+  template:
+    metadata:
+      name: stolon-backup
+    spec:
+      containers:
+      - name: backup
+        imagePullPolicy: Always
+        image: apiserver:5000/stolon-backup:0.0.1
+        command: ["/backup.sh"]
+        env:
+          - name: STOLON_BACKUP_DB
+            value: "{{STOLON_BACKUP_DB}}"
+        volumeMounts:
+          - mountPath: /backups
+            name: backups
+          - mountPath: /etc/secrets/stolon
+            name: stolon
+      volumes:
+        - name: backups
+          hostPath:
+            path: /var/backups
```

Comment:
its ok for dev mode, but we should have a capability to change it to something replicated like s3. So maybe it's better not to save it on local FS and pass through mount to host, but send immediately after backup to some storage